### DISCUSSION TITLE
Manual toggle of Accordion not working while using index

### DISCUSSION THREAD

**Discussion by creative-headr** [chakra-ui/chakra-ui#9390] [Q&A] [answered]
I have a group of nested accordions and there can be at max 3 sub levels of accordion within each outer accordion.
I am creating a functionality expand to L1, L2, L3, L4. Using this users will be able to open a specific level of all the accordions with a click of a button. I am maintaining a state and using index prop provided in Accordion, but doing so, I am losing the ability of manual toggle of accordion. 
How can I have both?
Or, is there any way to achieve this functionality without using index?

**Comment by segunadebayo** [comment]
Do you have a reproduction?

**Comment by segunadebayo** [comment, marked as answer]
`index` is v2. v3 Accordion is controlled with `value` and `onValueChange`.

Keep a `string[]` of open item values. Update that array from your L1/L2 buttons and from item clicks. Users can still toggle by hand.
